src/core/Akka.Remote.Tests/RemoteConsistentHashingRouterSpec.cs

```diff
@@ -45,6 +45,36 @@ public void ConsistentHashingGroup_must_use_same_hash_ring_independent_of_self_a
             var result2 = keys.Select(k => consistentHash2.NodeFor(k).Routee);
             Assert.Equal(result2,result1);
         }
+
+        [Fact]
+        public void ConsistentHashingGroup_must_use_same_hash_ring_independent_of_local_and_remote_nodes()
+        {
+            var a1 = new Address("akka.tcp", "RemoteConsistentHashingRouterSpec-1", "client1", 2552);
+            var a2 = new Address("akka.tcp", "RemoteConsistentHashingRouterSpec-1", "client2", 2552);
+            var localActor = Sys.ActorOf(Props.Empty, "a");
+
+            var s1 = new ActorRefRoutee(localActor);
```

Comment:
should add a comment here / something that makes it obvious how this reproduces the bug

Reply:
added it on the method